examples/src/main/java/alluxio/cli/TestRunner.java

```diff
@@ -45,6 +46,10 @@
   @Parameter(names = {"-h", "--help"}, description = "Prints usage information", help = true)
   private boolean mHelp;
 
+  @Parameter(names = "--directory",
+      description = "The directory to test (can be a non-root mount point). Default: /" )
```

Comment:
"The directory to write test files to"

It seems natural that `--directory` can be a non-root mount point, so it could be confusing that we explicitly call it out.